AGENTS.md

```
@@ -497,167 +358,186 @@ wegent/
 
 ### Executor
 
-**Tech:** Python, Claude Code SDK, Agno, Dify API, Docker
+**Tech:** Python, Claude Code SDK, Agno, Dify API, MCP
 
-**Supported Agent Types:**
-- **ClaudeCode**: For code development tasks with Claude Code SDK
-- **Agno**: For dialogue and chat tasks
-- **Dify**: For external API integration with Dify platform (chat, workflow, chatflow, agent-chat modes)
+**Agent types:**
+| Agent | Type | Key Features |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)